test/doctool/test-doctool-html.js

```diff
@@ -115,15 +115,15 @@ testData.forEach((item) => {
           const actual = output.replace(/\s/g, '');
           // Assert that the input stripped of all whitespace contains the
           // expected list
-          assert.notStrictEqual(actual.indexOf(expected), -1);
+          assert.notStrictEqual(actual.includes(expected), -1);
```

Comment:
This assertion will now never fire because `.includes()` returns `true` or `false`, both of which are not strictly equal to `-1`. I think ever line in this PR needs to be updated to remove the numeric comparison. Additionally, care must be taken to determine whether we need `foo.includes(bar)` vs. `! foo.includes(bar)`.